crates/astria-proto/proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -40,6 +41,7 @@ message Action {
 message TransferAction {
     bytes to = 1;
     astria.primitive.v1.Uint128 amount = 2;
+    optional bytes asset = 3;
```

Comment:
i don't think it's possible to do this - we might have arbitrary IBC assets transferred to us, so we don't know the entire potential limit of what we need to support ahead of time. this might help give more background: https://github.com/cosmos/ibc/blob/main/spec/app/ics-020-fungible-token-transfer/README.md i haven't implemented the ICS20 stuff yet, but it'll be the next step after this PR

and yes we can express all amounts this way, as the `asset` is an `ID` that points to a known base denomination of the asset. i'll rename this `asset_id` to make it clearer